src/adapters/bidderFactory.js

```diff
@@ -210,9 +249,14 @@ export function newBidder(spec) {
         }
       });
 
+      if (validBidRequests.length === 0) {
```

Comment:
I don't understand the logic why we only want to add no bids when there are no valid bids.  What's the thinking here?

Reply:
So if there is a bidderRequest.bids, but none of them are valid, this logs those bids as failed bids.  This is consistent with what would currently happen if a subset of the bids are invalid.  

If we want to only have no bids for valid requests we will have to use validBidRequests instead of bidderRequest.bids when looking for missing bids to maintain consistency.

Let me know what your preference is.  I am fine with it either way.